linux_os/guide/system/permissions/files/permissions_important_account_files/file_groupowner_backup_etc_gshadow/rule.yml

```diff
@@ -0,0 +1,36 @@
+documentation_complete: true
+
+title: 'Verify Group Who Owns Backup gshadow File'
+
+description: '{{{ describe_file_group_owner(file="/etc/gshadow-", group="root") }}}'
+
+rationale: |-
+    The <tt>/etc/gshadow-</tt> file is a backup of <tt>/etc/gshadow</tt>, and as such,
+    it contains group password hashes. Protection of this file is critical for system security.
+
+severity: medium
+
+identifiers:
+    cce@rhel7: 83534-8
+    cce@rhel8: 83535-5
+
+references:
+    cis@rhel7: 6.1.9
+    cis@rhel8: 6.1.9
+
+ocil_clause: '{{{ ocil_clause_file_group_owner(file="/etc/gshadow-", group="root") }}}'
+
+ocil: '{{{ ocil_file_group_owner(file="/etc/gshadow-", group="root") }}}'
+
+template:
+    name: file_groupowner
+    vars:
+        filepath: /etc/gshadow-
+        filegid: '0'
+        filegid@debian8: '42'
```

Comment:
I believe that if you make customizations per  product, this should be also reflected in description / ocil.